crates/nu-cli/tests/completions/mod.rs

```diff
@@ -2944,3 +2911,37 @@ fn type_inferenced_operator_completions(mut custom_completer: NuCompleter) {
     let expected: Vec<_> = vec!["=", "=="];
     match_suggestions(&expected, &suggestions);
 }
+
+#[test]
+fn flag_filepath_completions() {
+    use support::completions_helpers::new_engine_helper;
+    use tempfile::tempdir;
+
+    let test_dir = tempdir().unwrap();
```

Comment:
You don't need to make a temp directory, you can use one of the existing fixture files/directories. I guess the advantage of this is that it doesn't rely on those files, but that's what all the other tests use, so I'd prefer to go with that

Reply:
i'll mark this as resolve, just as i've pulled this part from the PR.